helm-chart/helm3/6.4.0/skywalking/templates/oap-deployment.yaml

```diff
@@ -91,6 +91,10 @@ spec:
               fieldPath: metadata.uid
         - name: SW_STORAGE
           value: elasticsearch
+        {{- if .Values.oap.envoy.als.enabled }}
```

Comment:
why do you add a parameter to control an env variable? all items listed in `.Values.oap.env` will be added here. Take a look at line 100.

Reply:
In order to enable envoy als, there is a clusterrole privilege, and when it is not enabled, only role privilege